content/influxdb/v2.1/security/enable-hardening.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Enable security features
+seotitle: Enable security/hardening features
+description: >
+  Enable additional security features in InfluxDB.
+weight: 102
+menu:
+  influxdb_2_1:
+    parent: Security & authorization
+influxdb/v2.1/tags: [security, hardening]
+products: [oss]
+---
+
+InfluxDB is used in many environments and depending on your environment, enabling additional security features may be appropriate.
+To enable all security hardening options now and in the future,
+start InfluxDB with the `--hardening-enabled` command line flag:
+
+    ```bash
+    influxd \
+    --hardening-enabled
+    ```
```

Comment:
```suggestion
InfluxDB {{< current-version >}} provides optional security features that ensure your
InfluxDB instance is secure in whatever environment it's used in.

To enable all [additional security features](#security-features), use the
[`hardening-enabled` configuration option](/influxdb/v2.1/reference/config-options/#hardening-enabled)
when starting InfluxDB.
```

There are three different ways to configure `influxd`; the CLI flag is only one. In the Configuration options doc, we show all three methods for setting config options. This also brings up additional changes that need to be made:

- Add a `hardening-enabled` section to the configuration options doc (`content//influxdb/v2.1/reference/config-options/`). Because this isn't available in 2.1 yet, we need to leave it commented out until 2.1 gets patched. We can port the change to the `oss-2.2-base` branch and uncomment the section.
- Update `data/influxd_flags.yml` with the new flag:

    ```yaml
    - flag: "--flux-log-enabled"
      added: 2.0

    - flag: "--hardening-enabled"
      added: 2.2

    - flag: "-h, --help"
      added: 2.0
      nolink: true
    ```

Reply:
> * Add a `hardening-enabled` section to the configuration options doc (`content//influxdb/v2.1/reference/config-options/`). Because this isn't available in 2.1 yet, we need to leave it commented out until 2.1 gets patched. We can port the change to the `oss-2.2-base` branch and uncomment the section.

@sanderson - what is the recommended way to comment out things? I read https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md but didn't see anything (maybe I missed it?).

Reply:
@jdstrand Markdown just uses HTML comments `<!--  -->`.

Oh, this reminds me... don't comment out the line item in middle of the the TOC list. It won't work. For now, just add it as a comment below the list with a note to move it into the list once 2.1 gets patched.